src/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs

```diff
@@ -104,15 +105,35 @@ public void OpenFile_ThrowsIOException()
             }
         }
 
-        [ActiveIssue(4605, PlatformID.OSX)]
```

Comment:
Actually, I'm wrong, I have not verified it on OSX. I didn't realize that the CI wasn't doing OSX verification.

Is there any somewhat simple way to verify without actually having an OSX machine?